src/main/java/seedu/address/model/account/Name.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.account;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an Account's name in the account book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
+ */
+public class Name {
```

Comment:
Both kenneth and you are using the name class. Perhaps extract it out into the main model package

Reply:
Our name class is different tho, my name class is account's name class, his name class represents person's name.

Reply:
Jiehan's correct, our Name classes are in different packages. Mine is in ```model/person/loan```.

Reply:
I meant that they have the same abstraction. We should keep the code DRY.

Reply:
Oh, in that case I see what you mean. The only concern might be that in the future Account/Loan name requirements might differ. I don't really see why that would happen, but I'll leave it up to Jiehan.